R/pkg/R/functions.R

```diff
@@ -1253,19 +1256,6 @@ setMethod("quarter",
             column(jc)
           })
 
-#' @details
-#' \code{reverse}: Reverses the string column and returns it as a new string column.
-#'
-#' @rdname column_string_functions
-#' @aliases reverse reverse,Column-method
-#' @note reverse since 1.5.0
-setMethod("reverse",
-          signature(x = "Column"),
-          function(x) {
-            jc <- callJStatic("org.apache.spark.sql.functions", "reverse", x@jc)
-            column(jc)
-          })
-
```

Comment:
so these func are not actually ordered or grouped - my preference would be not to move then around unless we have to?

Reply:
@felixcheung Thanks for your comment. The reason I moved reverse and concat around is that I want to put them under line 2943 for Collection functions. 
###################### Collection functions######################

Reply:
right, but that's not strictly where collection functions are.
I'm not saying we need to re-sort/group all functions - that will be thousands of lines we need to change

Reply:
I will put them back to the original places and add back the examples. Thanks!